ParkhomenkoOA/lab_1/main.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <algorithm>
+
+using namespace std;
+
+vector<string> inputWords;
+vector<string> wordsSums;
+vector<string> inputMarks;
+
+char dictionaryOfMarks[4] = { '.', ',', '!', '?' };
```

Comment:
You can extract '4' as named constant and use it over your program. Otherwise if I say that ':' is also a punctuation mark - you will have to change a lot of places in your code